bazel/envoy_build_system.bzl

```diff
@@ -117,14 +117,17 @@ def envoy_cmake(
         name,
         cache_entries = {},
         debug_cache_entries = {},
+        default_cache_entries = {"CMAKE_BUILD_TYPE": "Bazel"},
         lib_source = "",
         postfix_script = "",
         copy_pdb = False,
         pdb_name = "",
         cmake_files_dir = "$BUILD_TMPDIR/CMakeFiles",
         generate_crosstool_file = False,
+        generate_args = ["-GNinja"],
+        targets = ["", "install"],
         **kwargs):
-    cache_entries.update({"CMAKE_BUILD_TYPE": "Bazel"})
```

Comment:
not sure why this is enforced - i think in the case of vpp we _want_ it to be `Release`

Reply:
i'd have to dig up the history, there was a reason